plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -47,14 +60,17 @@ type operationNamesReader func(service string) ([]string, error)
 
 // SpanReader can query for and load traces from Cassandra.
 type SpanReader struct {
-	session                   cassandra.Session
-	consistency               cassandra.Consistency
-	serviceNamesReader        serviceNamesReader
-	operationNamesReader      operationNamesReader
-	readTracesTableMetrics    *casMetrics.Table
-	queryTraceTableMetrics    *casMetrics.Table
-	queryTagIndexTableMetrics *casMetrics.Table
-	logger                    *zap.Logger
+	session                           cassandra.Session
+	consistency                       cassandra.Consistency
+	serviceNamesReader                serviceNamesReader
+	operationNamesReader              operationNamesReader
+	readTracesTableMetrics            *casMetrics.Table
+	queryTraceTableMetrics            *casMetrics.Table
+	queryTagIndexTableMetrics         *casMetrics.Table
+	queryDurationIndexTableMetrics    *casMetrics.Table
+	serviceOperationIndexTableMetrics *casMetrics.Table
+	serviceNameIndexTableMetrics      *casMetrics.Table
```

Comment:
these need to be encapsulated into a nested metrics struct, no reason to have them all at the top level